rotkehlchen/tests/conftest.py

```diff
@@ -241,7 +242,19 @@ def vcr_cassette_dir(request: pytest.FixtureRequest) -> str:
             log.debug('VCR setup completed')
         else:
             log.debug('Pulling information for VCR cassettes')
-            os.popen(f'cd {tests_caching_dir} && git pull').read()
+            current_branch = os.popen('git rev-parse --abbrev-ref HEAD').read().rstrip('\n')
+            checkout_proc = Popen(f'cd {tests_caching_dir} && git checkout {current_branch}', shell=True, stdout=PIPE, stderr=PIPE)  # noqa: E501
+            _, stderr = checkout_proc.communicate(timeout=10)
+
+            if (
+                len(stderr) != 0 and
+                b'Already on' not in stderr and
+                b'Switched to branch' not in stderr
+            ):
+                default_branch = os.environ.get('DEFAULT_VCR_BRANCH', 'bugfixes')
```

Comment:
Defaulting to develop always would probably be best as I think the majority of PRs in the long run are against develop.

Also ... couldn't you somehow find the "parent" branch of a given branch? That would be the perfect solution to choose here automatically. https://stackoverflow.com/questions/3161204/how-to-find-the-nearest-parent-of-a-git-branch

I tried the solution listed there but got the `cannot handle more than 26 refs` so I did not look further.

Reply:
I saw that approach but for me it failed also with a similar error. I believe that it fails for the people using the tool then is not good